server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const path = require('path')
+const PORT = process.env.PORT || 3000
+
+const app = express()
+
+const index = require('./routes/index')
+
+app.use(express.static('public'))
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true }))
```

Comment:
Check whether you need `urlencoded`: if the client folks are sending JSON, you could remove this.